dbt_coves/tasks/generate/base.py

```diff
@@ -93,11 +98,27 @@ def get_relations(self, filtered_schemas):
                     rel_name_selectors.append(rel.name)
                     break
 
+        excluded = []
```

Comment:
It's missing the same transformation done in lines 86 to 89, replacing "*" by ".*"